06-foundational-skills_2.Rmd

```diff
@@ -608,6 +608,8 @@ It's common in the R programming world to name a package by choosing a word that
 
 In this case, the author of the {magrittr} package created a series of pipe operators and then collected them in a package named after the artist Magritte.
 
+R version 4.1.0 introduced the native pipe operator, `|>`. Like the {magrittr} pipe, it passes an object to a function so that you can chain a sequence of operations together. For straightforward use cases, both pipes behave similarly. However, you may run into differences in more complex scenarios. Since the {magrittr} pipe is tightly integrated with the tidyverse, we continue to use it in our code examples.
```

Comment:
Can we make this change to remove the first person POV please? From: 

> Since the {magrittr} pipe is tightly integrated with the tidyverse, we continue to use it in our code examples.

To: 

> The code examples in this book use the {magrittr} pipe because it is tightly integrated with the tidyverse.